packages/react-storage/src/components/StorageBrowser/controls/__tests__/DataRefreshControl.spec.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { useResolvedComposable } from '../hooks/useResolvedComposable';
+import { useDataRefresh } from '../hooks/useDataRefresh';
+import { DataRefreshControl } from '../DataRefreshControl';
+
+jest.mock('../hooks/useDataRefresh');
+jest.mock('../hooks/useResolvedComposable');
+
+describe('DataRefreshControl', () => {
+  // assert mocks
+  const mockUseDataRefresh = useDataRefresh as jest.Mock;
+  const mockUseResolvedComposable = useResolvedComposable as jest.Mock;
```

Comment:
Prefer spies over mocks. Also don't think we should mock `useResolvedComposable` since it returns the component you are testing?

Reply:
thx updated